README.md

```diff
@@ -69,3 +69,9 @@ operational burden.
 
 Further, to solve operational issues using the v3 API will enable us to use the
 backup features of the etcd Operator.
+
+# Run Cloudbuild
```

Comment:
Why do we want to run cloudbuild?  What does it do? We should have some words here about why. Something like, 

> Cloudbuild can be used to build and deploy discoveryserver.  `cloudbuild.dev.yaml` builds discoverserver, publishes the docker image, and deploys to the dev environment. `cloudbuild.prod.yaml` deploys a built docker image to the production environment.